src/tools/perf/lib/ucp_tests.cc

```diff
@@ -41,7 +42,23 @@ class ucp_perf_test_runner {
         m_max_outstanding(m_perf.params.max_outstanding)
 
     {
+        ucs_status_t status;
+        ucp_am_handler_param_t param;
```

Comment:
[minor] this is c++ code, we can declare variables at the place where we need them
wdyt?